x-pack/plugins/alerts/server/plugin.ts

```diff
@@ -129,6 +129,16 @@ export class AlertingPlugin {
     this.spaces = plugins.spaces?.spacesService;
     this.security = plugins.security;
 
+    core.capabilities.registerProvider(() => {
```

Comment:
I'm curious - we don't have one of these stanzas for actions, but I'm guessing we can only `registerProvider()` in one place.  What happens if security is enabled, alerts is disabled and actions is enabled?  Thinking about a future-Kibana that uses actions in other scenarios besides alerts ...

Reply:
Oh, actually, this might be an overwsigt.
@legrego do we still need this?

Reply:
Hmm I've checked and removing this causes it to appear to unprivileged users.
I think this PR might not be ready yet and might require some work by Security.

Reply:
Thanks for the ping - we still want this in place, at least for the time being. There is a bug in Feature Controls where features can't successfully "register" their management sections, because the `features` plugin isn't accounting for them when building the canonical set of Capabilities. The workaround is for plugins to provide this capability manually, until we can update the Features plugin accordingly.

This provider does not influence or control any sort of access management. It merely defines the boolean flag (with the correct default value of `true`) so that it may be controlled by Security/Spaces as needed.

FWIW, we can remove this once https://github.com/elastic/kibana/pull/67791 merges, as this PR updates the `features` plugin to respect management sections.